src/views/ModelPlan/TaskList/Basics/Milestones/index.tsx

```diff
@@ -156,436 +157,455 @@ const Milestones = () => {
       <BreadcrumbBar variant="wrap">
         <Breadcrumb>
           <BreadcrumbLink asCustom={Link} to="/">
-            <span>{generalT('home')}</span>
+            <span>{miscellaneousT('home')}</span>
           </BreadcrumbLink>
         </Breadcrumb>
         <Breadcrumb>
           <BreadcrumbLink asCustom={Link} to={`/models/${modelID}/task-list/`}>
-            <span>{generalT('tasklistBreadcrumb')}</span>
+            <span>{miscellaneousT('tasklistBreadcrumb')}</span>
           </BreadcrumbLink>
         </Breadcrumb>
-        <Breadcrumb current>{planBasicsMiscT('breadcrumb')}</Breadcrumb>
+        <Breadcrumb current>{basicsMiscT('breadcrumb')}</Breadcrumb>
       </BreadcrumbBar>
       <PageHeading className="margin-top-4 margin-bottom-1">
-        {planBasicsMiscT('heading')}
+        {basicsMiscT('heading')}
       </PageHeading>
       <p
         className="margin-top-0 margin-bottom-1 font-body-lg"
         data-testid="model-plan-name"
       >
-        {generalT('for')} {modelName}
+        {miscellaneousT('for')} {modelName}
       </p>
+
       <p className="margin-bottom-2 font-body-md line-height-sans-4">
-        {generalT('helpText')}
+        {miscellaneousT('helpText')}
       </p>
 
       <AskAQuestion modelID={modelID} />
-      <Formik
-        initialValues={initialValues}
-        onSubmit={() => {
-          handleFormSubmit();
-        }}
-        enableReinitialize
-        validateOnBlur={false}
-        validateOnChange={false}
-        validateOnMount={false}
-        innerRef={formikRef}
-      >
-        {(formikProps: FormikProps<InitialValueType>) => {
-          const {
-            errors,
-            handleSubmit,
-            setErrors,
-            setFieldError,
-            setFieldValue,
-            validateForm,
-            values
-          } = formikProps;
-          const flatErrors = flattenErrors(errors);
-
-          const handleOnBlur = (
-            e: React.ChangeEvent<HTMLInputElement>,
-            field: string
-          ) => {
-            if (e.target.value === '') {
-              setFieldValue(field, null);
-              return;
-            }
-            try {
-              setFieldValue(field, new Date(e.target.value).toISOString());
-              delete errors[field as keyof InitialValueType];
-            } catch (err) {
-              setFieldError(field, generalT('validDate'));
-            }
-          };
-          return (
-            <>
-              {Object.keys(errors).length > 0 && (
-                <ErrorAlert
-                  testId="formik-validation-errors"
-                  classNames="margin-top-3"
-                  heading={generalT('checkAndFix')}
+
+      {!loading && (
```

Comment:
Can you add a comment clarifying the importance/purpose of this `!loading` check?